editor/components/post-publish-panel/prepublish.js

```diff
@@ -12,7 +19,18 @@ import PostVisibilityLabel from '../post-visibility/label';
 import PostSchedule from '../post-schedule';
 import PostScheduleLabel from '../post-schedule/label';
 
-function PostPublishPanelPrepublish() {
+function PostPublishPanelPrepublish( {
+	hasPublishAction,
+} ) {
+	if ( ! hasPublishAction ) {
+		return (
+			<div className="editor-post-publish-panel__prepublish">
+				<div><strong>{ __( 'Are you ready to submit for review?' ) }</strong></div>
+				<p>{ __( 'Here, you can do a last-minute check up of your settings below, before you submit for review.' ) }</p>
+			</div>
+		);
+	}
```

Comment:
It feels that a lot of things are shared with the code bellow. Maybe we can just do something like:

 `{ hasPublishAction ? __( 'Are you ready to publish?' ) :  __( 'Are you ready to submit for review?' ) }`

And wrap the panels inside a 
```js
{ hasPublishAction && (
<Fragment>
  // panels go here
</Fragment>
) }
```